### PR TITLE
Create is31fl3741_13x9_sequential_chain.py

### PR DESCRIPTION
**Pull request by DJDevon3**
An example using 4 chained 13x9 matrix's using STEMMA. Possibly due to power requirements and/or inability to setup a tuple for addresses currently could only get them to run sequentially. A cheap trick that will really only work for scrolling marquee. It's still a valid example though.

Have to initialize each matrix manually because currently the address field will not accept a tuple of i2c addresses, only an int.

https://user-images.githubusercontent.com/49322231/231340376-ab9ef04d-d8c7-40c1-a169-b3acafcef1f3.mp4

edit: github doesn't like .mov format, converted it to .mp4

### CODE REVIEW

**Comment by DJDevon3** [comment]
I'm using `Matrix31.global_current = 0x01` at its lowest possible setting of 1 (out of 255).  Using the default 0xFF current value of 255 does work sequentially at full brightness because only 1 matrix is in use at a time. Cheap work around for now.  

Really need tuple address support to test the power consumption limitations of simultaneous chaining and enlarging the matrix to 52x9 horizontally (because these are obviously designed to only be chained horizontally).

**Review by gamblor21 — COMMENTED** [review]
I only have one 13x9 matrix so cannot test the example but it looks good to me reading it over. I'll see if anyone else that has it wants to test before approval.

**Review by gamblor21 — APPROVED** [review]
Tested this on 3 matrices and it works. I was able to increase the global current higher but at 0x01 it works fine.

**Pull request merged**
